sdk/test/AzureTest/DVRSessionTransport.swift

```diff
@@ -58,7 +58,9 @@ public class DVRSessionTransport: TransportStage {
 
     public func open() {
         guard session == nil else { return }
-        session = Session(cassetteName: cassetteName)
+        let sdkPath = environmentVariable(forKey: "SDK_REPO_ROOT", default: "~")
+        let outputDirectory = sdkPath + ((sdkPath.last == "/") ? "sdk/communication/AzureCommunicationChat/Tests/Util/Recordings" : "/sdk/communication/AzureCommunicationChat/Tests/Util/Recordings")
```

Comment:
`<MOD>/Tests/Recordings` is probably a sufficient location.

Reply:
Also, instead of this, you should be able to use `sdkPath.appendingPathComponent` and not have to do this special checking of separators.

Reply:
When you say <MOD>, do you mean the location pointed to by the env variable?

Reply:
Well, it would be:
`<SDK_PATH>/sdk/communication/<MODULE>/Tests/Recordings`

In other words, recordings don't need to be in Util. :)

Reply:
sdkPath is a string, so I can't use that function. I did not convert it into a URL because outputDirectory takes a string and the environment variable gives out a string as well, I thought it'd be more tedious to convert to a URL just to go back to a string.

Reply:
It's better to go to URL and manipulate that way and then go back to string. Duplicating the path is a red flag. If somewhere else you forget to do this, you have bugs. If you temporarily convert to URL, manipulate and then go back, you are good. 
```swift
guard let outputDirectory = URL(sdkPath)?.appendingPathComponent("sdk/blah/")?.absoluteString else { error("SDK path invalid") }
```